src/ng/directive/ngModel.js

```diff
@@ -1215,7 +1215,7 @@ var ngModelOptionsDirective = function() {
     restrict: 'A',
     controller: ['$scope', '$attrs', function($scope, $attrs) {
       var that = this;
-      this.$options = $scope.$eval($attrs.ngModelOptions);
+      this.$options = angular.copy($scope.$eval($attrs.ngModelOptions));
```

Comment:
You are right, should be able to make the change in about an hour (unless
someone beats me)
>
>